test/iobuffer.jl

```diff
@@ -74,6 +74,12 @@ Base.compact(io)
 @test_throws ArgumentError seek(io,0)
 @test_throws ArgumentError truncate(io,0)
 @test readline(io) == "whipped cream\n"
+@test write(io,"pancakes\nwaffles\nblueberries\n") > 0
```

Comment:
Could you add a few non-ASCII characters like `ë` and `€`? This would ensure we don't assume 1-byte characters.